sdk/guides/github-workflows/assign-reviews.mdx

```diff
@@ -1,30 +1,72 @@
 ---
 title: Assign Reviews
-description: Automatically assign relevant reviewers to pull requests using OpenHands Agent
+description: Automate PR management with intelligent reviewer assignment and workflow notifications using OpenHands Agent
 ---
 
 <Note>
-This example is available on GitHub: [.github/workflows/assign-reviews.yml](https://github.com/OpenHands/agent-sdk/blob/main/.github/workflows/assign-reviews.yml)
+This example is available on GitHub: [examples/03_github_workflows/01_basic_action/assign-reviews.yml](https://github.com/OpenHands/agent-sdk/blob/main/examples/03_github_workflows/01_basic_action/assign-reviews.yml)
 </Note>
 
-Automatically assign relevant reviewers to pull requests based on code changes and file ownership using OpenHands Agent.
-This workflow uses the basic action workflow template that allows to perform any basic action with the OpenHands Agent. 
+Automate pull request triage by intelligently assigning reviewers based on git blame analysis, notifying reviewers of pending PRs, and prompting authors on stale pull requests. The agent performs three sequential checks: pinging reviewers on clean PRs awaiting review (3+ days), reminding authors on stale PRs (5+ days), and auto-assigning reviewers based on code ownership for unassigned PRs.
+
+## How it works
+
+It relies on the basic action workflow (`01_basic_action`) which provides a flexible template for running arbitrary agent tasks in GitHub Actions.
+
+**Core Components:**
+- **`agent_script.py`** - Python script that initializes the OpenHands agent with configurable LLM settings and executes tasks based on provided prompts
```

Comment:
not blocking: I was actually wondering if we can make agent_script generic enough and put it into one overview, and we can create different workflows (eg assign reviews, fix todos) based on different prompt but refer to the same agent_script?